expired_sstables.py

```diff
@@ -0,0 +1,543 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020 ScyllaDB
+#
+# Author: Raphael S. Carvalho <raphaelsc@scylladb.com>
+#
+# Inherited murmur3_token from Cassandra's python-driver
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+import os
+import subprocess
+import sys
+import time
+import argparse
+import struct
+import psutil
+
+# WARNING: don't tweak the value below whatsoever as it's a CONSTANT.
+deletion_time_for_never_expiring_sstable = 2147483647 # timestamp in seconds
+
+class inclusive_range:
```

Comment:
Better to use Interval from the intervaltree module.

Reply:
will change in next version. thanks